packages/entity-list/src/modules/preferences/sagas.js

```diff
@@ -81,3 +87,53 @@ export function* resetPreferences() {
   yield call(listSagas.setSorting)
   yield call(listSagas.reloadData)
 }
+
+export function* displayColumnModal() {
+  const {formDefinition} = yield select(listSagas.listSelector)
+  const {columns: preferencesColumns} = yield select(preferencesSelector)
+  const formColumns = formDefinition.children
```

Comment:
could be moved to util file:
https://github.com/tocco/tocco-client/blob/master/packages/entity-list/src/util/api/forms.js
And use "getTable" from there